posts/2020-05-15-six-years-of-rust.md

```diff
@@ -0,0 +1,71 @@
+
+Rustc wasn't the only tool that had significant improvements.
+
+- Cargo just recently stabilised its new feature resolver, that makes it easier to use your dependencies across different targets.
+
+- Rustdoc stabilised its "intra-doc links" feature, allowing you to easily and automatically cross reference Rust types and functions in your documentation.
+
+
+## Rust In Production
+Each year Rust's growth and adoption in the community and industry has been unbelievable, and this past year has been no exception. Once again for the sixth year in a row, Rust has been voted Stackoverflow's Most Loved Programming Language.
+
+With the formation of the [Rust foundation], Rust has been in a better position build a sustainable open source ecosystem empowering everyone to build reliable and efficient software.
+
+And it isn't just Rust that has been getting bigger. Larger and larger companies have been adopting Rust in their projects and offering officially supported Rust APIs.
+
+- Both Microsoft and Amazon have just recently announced and released their new officially supported Rust libraries for interacting with [Windows] and [AWS]. Official first party support for these massive and monolithic APIs helps make Rust people's first choice when deciding what to use for their project.
+- The cURL project has released new versions that offer opt-in support for using Rust libraries for handling [HTTP/s] and [TLS] communication. This has been a huge inter-community collaboration between the ISRG, the Hyper & Rustls teams, and the cURL project, and we'd like to thank everyone for their hard work in providing new memory safe backends for a project as massive and widely used as cURL!
+- Tokio (an asynchronous runtime written in Rust), released its [1.0 version][tokio-1.0] and announced their three year stability guarantee, providing everyone with a solid, stable foundation for writing reliable network applications without compromising speed.
+
+[tokio-1.0]: https://tokio.rs/blog/2020-12-tokio-1-0
+[http/s]: https://daniel.haxx.se/blog/2020/10/09/rust-in-curl-with-hyper/
+[tls]: https://daniel.haxx.se/blog/2021/02/09/curl-supports-rustls/
+[rust foundation]: https://foundation.rust-lang.org/posts/2021-02-08-hello-world/
+[windows]:https://github.com/microsoft/windows-rs
+[aws]: https://github.com/awslabs/aws-sdk-rust
+
+## Future Work
+Of course, all that is just to start, we're seeing more and more initiatives putting Rust in exciting new places;
+
+- Critical Section & Ferrous Systems have started [Ferrocene]. A project to make Rust a viable programming language for safety and mission critical systems across the industry.
+- Embark Studios have released an initial prototype of [`rust-gpu`], a new compiler backend that allows writing graphics shaders using Rust for GPUs.
```

Comment:
```suggestion
- Embark Studios have released an initial prototype of [`rust-gpu`], a new compiler backend that allows writing graphics shaders using Rust for GPUs, and is now [using it in production for 100% of their shaders](https://mailchi.mp/embark-studios/dev009)
```

(Feel free to split the link to a footnote, something like "rust-gpu-in-production".)

Reply:
(Closing this based on discussion with Embark folks.)